extensions/wikia/RTE/RTE.class.php

```diff
@@ -483,7 +477,7 @@ public static function getEdgeCaseType() {
 	/**
 	 * Parse given wikitext to HTML for CK
 	 */
-	public static function WikitextToHtml($wikitext) {
+	public static function WikitextToHtml( $wikitext ) {
 		global $wgTitle;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)